packages/app/error-utils/package.json

```diff
@@ -1,29 +1,22 @@
 {
 	"name": "@lokalise/error-utils",
-	"version": "1.2.0",
-	"type": "module",
+	"version": "1.2.1",
```

Comment:
I thin version will be updated automatically on release action

Reply:
true. fixed!